mitodl/mitxonline
# Update justifications based on action in refine, allow it to be blank for reset

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/mitxonline/issues/956

#### What's this PR do?
Updates **justification** options based on the user's action for the financial assistance request in the Staff dashboard
- For "Approve", 'Document In Order" is the only option, no matter what the prior state is.
- For "Deny",  "Documents in order" is removed as an option
- For "Reset",  blank is the default option, no matter what the prior state is.

Creates a migration to allow the justification field as blank in the FlexiblePricing model (Otherwise it can't be submitted with blank for reset)

Adds a few constants for flexible pricing justification


#### How should this be manually tested?

- Go to Flexible Pricing Requests on the staff dashboard: http://mitxonline.odl.local:8013/staff-dashboard/flexible_pricing
- Click on "Approve",  `Documents in order` should be the only option
- Click on "Deny",  `Documents in order` should not be an option
- Click on "Reset", It should default to blank and can be submitted as blank

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/3138890/190010454-17d8836b-9f01-468f-a830-1a2fae53bcca.png)

![image](https://user-images.githubusercontent.com/3138890/190010556-04e3df1c-5a20-4c20-9db5-a838155b7758.png)

![image](https://user-images.githubusercontent.com/3138890/190010702-da20eff9-f21c-4080-b92d-f507e7fd7deb.png)
